docs/index.rst

```diff
@@ -595,6 +678,50 @@ Users
        "last_name": "Watson"
      }
 
+.. http:post:: /password-resets/
```

Comment:
Can't remember what we discussed that well tbh :) what is ``http:create``?

I vaguely remember us discussing using different verbs for the same api endpoint, one for the request, one for the confirmation, but that would require us providing data structures that are not alike for each verb, which doesn't seem very RESTful.

My current approach seemed the closest to RESTful of the ideas I thought of and of approaches I read up on:
  - creating a password reset resource
  - creating a password reset confirmation resource